### DISCUSSION TITLE
Tool selection actions

### DISCUSSION THREAD

**Discussion by Geant200** [Zrips/CMI#7837] [Q&A]
Good morning

I'm not posting in the right place? Before asking my problem I wanted to say congratulations for your plugin ^^.

My problem is the selection tools which conflict with the RampDrill plugin.

https://www.spigotmc.org/resources/rampendrills.23086/

In the RampDrill plugin we use the tool (wooden hoe) to start the drill.

I have already changed the tool in RampDrill but it's always the CMI tool that takes over RampDrill I can't use the RampDrill tool function to manage the drill.

How can I make the tool I have configured in RampDrill work without CMI interfering with the operation of RampDrill.

I have already changed the tool in each plugin, it's always CMI that does the action but not RampDrill.

I thank you in advance.

**Comment by Geant200** [comment]
Can you help me please

**Comment by CloudeLecaw** [comment]
you know that you can define any item as selection tool? the woode shovel is just the default item..
so go to config.yml and change it to any item you like

```
Selection:
  # Tool id to use for selection actions
  Tool: wooden_shovel
```

so simply use different tools for each plugin

cmi normaly uses no wodden hoe..so i dont see what you may made wrong.... pls check what plugin you else are using is using the wooden hoe.

**Comment by Geant200** [comment]
Thank you for your answer I tested this method I apply them to your plugin and also to the other RampDrill plugin if I want the RampDrill plugin to work I must deactivate your plusgin even when I make the tool changes your plugin takes the above on the RampDrill plugin.

Can you do the test with your plugin and RampDrill change the tools you will find that the plugin does not work and that your plugin takes over RampDrill every time ^^.

Thanks in advance.

**Comment by Geant200** [comment]
an idea ? ^^.

**Comment by Geant200** [comment]
Good morning

I just created some videos to show you what I'm talking about.

With Cmi and RampDrill
https://www.war-team.com/autre/minecraft/avec_cmi_1.mp4

Without Cmi and RampDrill
https://www.war-team.com/autre/minecraft/sans_cmi_1.mp4

Cmi Setup - wooden shovel
https://www.war-team.com/autre/minecraft/cmi.jpg

Ramdrill Setup - wooden hoe
https://www.war-team.com/autre/minecraft/rampdrill.jpg

**Comment by Geant200** [comment]
have you looked at the videos and images to help me find a solution please.

Thanks in advance.

**Comment by Geant200** [comment]
any idea please?
